builtin/providers/cloudstack/resource_cloudstack_loadbalancer_test.go

```diff
@@ -44,6 +45,14 @@ func TestAccCloudStackLoadBalancerRule_update(t *testing.T) {
 				Config: testAccCloudStackLoadBalancerRule_basic,
 				Check: resource.ComposeTestCheckFunc(
 					testAccCheckCloudStackLoadBalancerRuleExist("cloudstack_loadbalancer_rule.foo"),
+					func(s *terraform.State) error {
+						rs, ok := s.RootModule().Resources["cloudstack_loadbalancer_rule.foo"]
+						if !ok {
+							return fmt.Errorf("Not found: cloudstack_loadbalancer_rule.foo")
+						}
+						id = rs.Primary.ID
+						return nil
+					},
```

Comment:
Get it... Will merge this as is, but will refactor it a little to make it fit in better with the existing code. Thx!